### ISSUE TITLE
Wifi Settings Page

### ISSUE THREAD

**Issue by alexanderkjones** [alexanderkjones/GM_V1#4]
**Story**
As a user, I want to see my current wifi settings and change them, in order to confirm my current settings and make adjustments if needed.

**Desc**
After waking the device, the user will connect to it over wifi using the AP and navigating to 192.168.4.1 in their browser. Their primary purpose will be to confirm wifi connection integrity. The device should attempt to connect to wifi once woken for configuration to confirm realtime connection status and populate display described below. If the user changes the wifi settings the device should attempt to connect using the new router settings to confirm settings are correct.

**Definitions**

[CONFIGURED SSID] - The SSID stored for connection settings

[LAST CONNECTION DATE/TIME] - Last time the device successfully received a 200 from the server while logging OR connection time at wake for config if it was successful.

[CONNECTION STRENGTH] - The strength of the connection at last connection time if not currently connecting or current connection strength if connecting successfully. Please use these strength definitions: http://luxul.com/wp-content/uploads/signal-strength.jpg

**Open Questions**
- Do we have room in program memory to utilize CSS?

**Copy**

If the device successfully connected during config startup
```
<h1>Your Wifi Settings<h1>
You are currently configured to connect to [CONFIGURED SSID]
Your last successful connection was [LAST CONNECTION DATE/TIME]
Your router's connection strength is [CONNECTION STRENGTH]
```

If device was unable to connect during config startup
```
<h1>Your Wifi Settings<h1>
You are currently configured to connect to [CONFIGURED SSID]
Your device has not connected to since <span style="font-color:#D8000C; font-weight:bold"> [LAST CONNECTION DATE/TIME]</span>
Your router's last recorded connection strength was [CONNECTION STRENGTH]
```

Webform to show in either case
```
<h2>To Reconfigure, Enter a new Wifi Router Name and Password<h2>
<form>
  Router Name:<br>
  <input type="text" name="SSID"><br>
  Password:<br>
  <input type="text" name="PASS"><br>
 <input type="submit" value="Save">
</form>
```
**Acceptance Criteria**
- The user can see their current wifi connection status
- The user can change their SSID and PASS settings
- The user can see the effect on connection to router after they've changed SSID/PASS
- The device continues to connect to reconfigured wifi settings for subsequent log cycles

**Comment by alexanderkjones** [comment]
@JLJTGR eta on this issue?

**Comment by JLJTGR** [comment]
I had to diagnose some hardware issues before starting on this.  The second board that I received from Matt was somehow wired badly so that the reset line was permanently tied to ground, which prevented the WDT pulse from waking it back up.  So the second board with the hardware you wanted to use was non-functioning for your previous prerequisites.  Visually, I could not tell what was wrong, so I moved the new hardware to the old, otherwise working board. I put up a commit that adapted our previous EEPROM/FRAM helper functions to the chip on the RTC board once I figured out how to use it.

I'll probably be looking at it again on Monday.
